### ISSUE TITLE
Enable filters for torrents view

### ISSUE THREAD

**Issue by nayots** [erengy/taiga#968]
There is no valid way to make a filter that only shows torrents that have a Video quality of 1080p with the current application.

There should be an option to filter each column by regex.
![image](https://user-images.githubusercontent.com/10439945/106786774-9b61d500-6657-11eb-9950-7fa83d071576.png)

**Comment by erengy** [comment]
> make a filter that only shows torrents that have a Video quality of 1080p

**Conditions:**
Episode video resolution | is not | `1080p`

**Action:**
Discard matched items

**Discard type:**
Hide discarded items

---

If all you need is 1080p, you can specify it at your RSS feed URL rather than creating a filter. This will give you more results.

- `https://nyaa.si/?page=rss&c=1_2&f=0` (default RSS feed)
- `https://nyaa.si/?page=rss&c=1_2&f=0&q=1080p` (limited to 1080p)

**Comment by nayots** [comment]
This is not a solution since the q query param is used by the title placeholder and adding 1080p does not return only 1080p quality.
```
&q=%title%
```

**Comment by paulo27ms** [comment]
The feed with that is the one you use for searching. Just add 1080p to your search or change your filter so non-1080p torrents are hidden by default.

**Comment by nayots** [comment]
It just does not work
![image](https://user-images.githubusercontent.com/10439945/110144334-e69b0f00-7de0-11eb-8ab7-9afee20627db.png)

**Comment by nayots** [comment]
A simple regex filter for a column would be a simple but effective fix.

**Comment by paulo27ms** [comment]
It is working perfectly fine for me.

![image](https://user-images.githubusercontent.com/13546823/110145049-eabf3080-7dd0-11eb-8964-8a23cda35c39.png)

![image](https://user-images.githubusercontent.com/13546823/110145180-0fb3a380-7dd1-11eb-8103-2a5dd7014349.png)

FYI: If you're having issues with "is not 1080p", consider "does not contain 1080".
But your issue is 100% certain to be caused by another filter that you have that does discards but does not hide those torrents. Taiga doesn't discard torrents twice.

If I add this filter on top I have the same issue as you:

![image](https://user-images.githubusercontent.com/13546823/110145756-ad0ed780-7dd1-11eb-95a0-65d95e29c43a.png)

![image](https://user-images.githubusercontent.com/13546823/110145703-a08a7f00-7dd1-11eb-81b1-b998effe5c88.png)

Basically to fix your issue all you have to do is move that filter to the top of the list instead of at the bottom.

**Comment by nayots** [comment]
Moving the filter to the top has yielded the desired result. Thanks @paulo27ms
